dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -648,7 +655,10 @@ class ApplicationsControllerTest < ::ActionController::TestCase
       time = Date.new(2017, 3, 15)
 
       Timecop.freeze(time) do
-        workshop = create :workshop, :local_summer_workshop, num_sessions: 3, sessions_from: Date.new(2017, 1, 1), processed_location: {city: 'Orchard Park', state: 'NY'}.to_json
+        workshop = create :summer_workshop,
+          num_sessions: 3,
```

Comment:
I don't think there is a reason that this needs to have 3 sessions - it looks like it could be a standard 5-day summer workshop